tests/src/OneLogin/Saml2/UtilsTest.php

```diff
@@ -828,6 +828,9 @@ public function testQuery()
      */
     public function testGenerateNameIdWithSPNameQualifier()
```

Comment:
we have some failures for this test, pls have a look

Reply:
Done!